.github/workflows/releases.yml

```diff
@@ -123,23 +123,17 @@ jobs:
           unzip -o *.zip && rm -v *.zip
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-      - name: Install go-msi
-        run: choco install -y "go-msi"
       - name: Prepare PATH
-        shell: bash
-        run: |
-          echo "$WIX\\bin" >> $GITHUB_PATH
-          echo "C:\\Program Files\\go-msi" >> $GITHUB_PATH
+        uses: microsoft/setup-msbuild@v1.0.3
       - name: Build MSI
         id: buildmsi
         shell: bash
         env:
           ZIP_FILE: ${{ steps.download_exe.outputs.zip }}
         run: |
-          mkdir -p build
-          msi="$(basename "$ZIP_FILE" ".zip").msi"
-          printf "::set-output name=msi::%s\n" "$msi"
-          go-msi make --msi "$PWD/$msi" --out "$PWD/build" --version "${GITHUB_REF#refs/tags/}"
+          name="$(basename "$ZIP_FILE" ".zip")"
+          printf "::set-output name=msi::%s\n" "$name.msi"
+          msbuild .\build\windows\gh.wixproj /p:SourceDir="$PWD" /p:OutputPath="$PWD" /p:OutputName="$name" /p:ProductVersion="${GITHUB_REF#refs/tags/}"
```

Comment:
Done. I also got rid of that `printf` line since _gh.wixproj_ will do that with a full path. The only question is whether its backslashes will be a problem, though it's just a string passed to a command. The _.wixproj_ could always replace backslashes to forward slashes. The _.wixproj_ will always have a full path, so it seems safer to keep that logic consolidated.